docs/data/release_archive.yml

```diff
@@ -661,6 +661,11 @@ release_archive:
       version: 4.0.0
       release_date: "2024-11-26"
       filename: "hbase"
+    - name: "Flink MongoDB Connector"
+      connector: "mongodb"
+      version: 2.0.0
+      release_date: "2025-02-05"
```

Comment:
nit: Should be updated when the release will actually happen to reflect the actual release date.